cmd/bridge/main.go

```diff
@@ -30,11 +30,11 @@ const (
 	k8sInClusterCA          = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 	k8sInClusterBearerToken = "/var/run/secrets/kubernetes.io/serviceaccount/token"
 
-	// Well-known location of Prometheus service for OpenShift. This is only accessible in-cluster.
-	openshiftPrometheusHost = "prometheus-k8s.openshift-monitoring.svc:9091"
+	// Well-known location of Thanos service for OpenShift. This is only accessible in-cluster.
+	openshiftPrometheusHost = "thanos-querier.openshift-monitoring.svc:9091"
```

Comment:
ok, we just found an edge case with @kyoto with respect to alerts in the openshift console. It turns out that console is hitting the `/rules` endpoint from Prometheus to render alerts.

For these requests we have to still hit the cluster Prometheus endpoint. For this code this means, instead of rewriting the target service URL, we need to create a new variable pointing to thanos and:
1. execute all `/query` requests against the thanos service
2. execute all `/rules` requests agains the cluster prometheus-k8s service

This implies that only cluster alerts are visible in the openshift console, not user workload alerts, which i _believe_ is fine for tech preview.

/cc @sichvoge @brancz @kyoto

Reply:
Reason: thanos querier does not implement the `/rules` endpoint.

Reply:
`/rules` can't really be implemented by Thanos. It refers to which rules are loaded by the Prometheus. I think the right thing to do is request the `/rules` endpoint of the cluster-monitoring Prometheus on the admin console. Do we already show alerting in the dev console? If not then this might need a feature in `prom-label-proxy` as we'd want to filter the alerts by their namespace label and request them from the user workload monitoring one.

Reply:
@brancz afaik, alerts are only displayed on the cluster overview, but @kyoto might have more insight.

Reply:
In that case let's have it hit the cluster-monitoring one only until we introduce the alerting section in the dev console.

Reply:
sounds good :+1:

Reply:
We are not currently hitting the `/rules` endpoint for the developer perspective, only for the admin perspective.

In the admin perspective, we use `/rules` for both the dashboards and for the "Monitoring" -> "Alerting" page.